internal/github/github.go

```diff
@@ -29,6 +29,12 @@ type client struct {
 	ghc *github.Client
 }
 
+func NewTestClient(c *github.Client) Client {
+	return &client{
+		ghc: c,
+	}
+}
```

Comment:
As suggested in the other comment, this new func is probably not necessary if we can reuse the existing code base for adding this specific test case.

As a side note -- having these "test" only functions are certainly useful sometimes, and I'd suggest having this defined within `_test.go` instead. It would allow adding extra logic only available in internal testing only. 👍
(You could keep it here but lowercase the first char, so that the function is only available within the defined package -- but it would still be a bit confusing for someone to find a function that's specifically used only for testing 😉)